client/src/pages/AboutUs.js

```diff
@@ -377,6 +379,25 @@ const AboutUs = () => {
         </SupportersLogosContainer>
       </SupporterContainer>
 
+      <PressContainer>
+        <FlexBox direction="column" align="center">
+          <h1>{t("Press")}</h1>
```

Comment:
@chrislemus Please update the translation file for the corresponding ids. Use camelCase for Ids in en_US.json you will find `aboutUs` property, you can create your own property inside that...